plugins/docs-tools/README.md

```
@@ -0,0 +1,423 @@
+# Uses .agent_workspace/docs-quick.yaml
+/docs-orchestrator PROJ-123 --workflow quick
+
+# Uses .agent_workspace/docs-full.yaml
+/docs-orchestrator PROJ-123 --workflow full
+```
+
+### Merge request creation
+
+The default workflow includes a **create-merge-request** step that is off by default. Pass `--create-merge-request` to activate it. When enabled, the step creates a feature branch (if needed), commits the written files, pushes to the remote, and opens a merge request (GitLab) or pull request (GitHub).
+
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)